### DISCUSSION TITLE
Multi frame jpegs Load succeeds, Identify fails

### DISCUSSION THREAD

**Discussion by eelke-at-bolt** [SixLabors/ImageSharp#1220] [Q&A] [answered]
I was using Identify to extract with and height of the image. However on some images I get the error `SixLabors.ImageSharp.InvalidImageContentException: Multiple SOF markers. Only single frame jpegs supported.` However when I use load on the same image I do not get an error. I would expect both functions to either fail or succeed. Is this a bug or is there an explanation for this behavior?

**Comment by JimBobSquarePants** [comment, marked as answer]
Sounds like a bug. Could you raise an issue please supplying an image to test against. 👍

**Reply by eelke-at-bolt** [reply]
Yes, [issue](https://github.com/SixLabors/ImageSharp/issues/1221)

**Reply by JimBobSquarePants** [reply]
Thanks! will have a look asap
